shell/ci/release/docker.sh

```diff
@@ -164,6 +148,14 @@ build_and_push_image() {
         -t "$remote_image_name:$VERSION" -t "$remote_image_name:latest" --push \
         "$buildContext"
     )
+  else
+    # When we're not on a tag, just build the docker image to verify
+    # that it is buildable.
+    info "Building Test Docker Image"
+    (
+      set -x
+      docker buildx build "${args[@]}" -t "$image" --load "$buildContext"
```

Comment:
Any reason you're omitting `--platform`?

Maybe store common args in an array ... ?

Reply:
I should've expanded in a comment here. We omit `platform` because we want to do a quick validation. Though, I guess building for both platforms is a good idea, in retrospect.